hadoop-tools/hadoop-azure/src/main/java/org/apache/hadoop/fs/azurebfs/services/AbfsClient.java

```diff
@@ -258,9 +258,9 @@ private AbfsClient(final URL baseUrl,
         throw new IOException("Exception while initializing metric credentials " + e);
       }
     }
-    this.timer = new Timer(
-        "abfs-timer-client", true);
     if (isMetricCollectionEnabled) {
+      this.timer = new Timer(
+              "abfs-timer-client", true);
```

Comment:
While we are changing this, let's also define a Constant for the Thread name and use that here instead of hardcoded string. We can define constant in the same file as
`public static String ABFS_TIMER_THREAD_NAME = abfs-timer-client`